code/modules/reagents/chemistry/reagents/other_reagents.dm

```diff
@@ -1788,16 +1787,15 @@
 	color = "#A86B45" //hair is brown
 	taste_description = "sourness"
 
-/datum/reagent/barbers_aid/expose_mob(mob/living/M, method=TOUCH, reac_volume)
-	if(method == TOUCH || method == VAPOR)
-		if(M && ishuman(M) && !HAS_TRAIT(M, TRAIT_BALD))
-			var/mob/living/carbon/human/H = M
-			var/datum/sprite_accessory/hair/picked_hair = pick(GLOB.hairstyles_list)
-			var/datum/sprite_accessory/facial_hair/picked_beard = pick(GLOB.facial_hairstyles_list)
-			to_chat(H, "<span class='notice'>Hair starts sprouting from your scalp.</span>")
-			H.hairstyle = picked_hair
-			H.facial_hairstyle = picked_beard
-			H.update_hair()
+/datum/reagent/barbers_aid/expose_mob(mob/living/M, methods=TOUCH, reac_volume)
+	if((methods & (TOUCH|VAPOR)) && M && ishuman(M) && !HAS_TRAIT(M, TRAIT_BALD))
+		var/mob/living/carbon/human/H = M
+		var/datum/sprite_accessory/hair/picked_hair = pick(GLOB.hairstyles_list)
+		var/datum/sprite_accessory/facial_hair/picked_beard = pick(GLOB.facial_hairstyles_list)
+		to_chat(H, "<span class='notice'>Hair starts sprouting from your scalp.</span>")
+		H.hairstyle = picked_hair
+		H.facial_hairstyle = picked_beard
+		H.update_hair()
```

Comment:
```suggestion
	if((methods & (TOUCH|VAPOR)) && ishuman(M) && !HAS_TRAIT(M, TRAIT_BALD))
		var/mob/living/carbon/human/exposed_human = M
		var/datum/sprite_accessory/hair/picked_hair = pick(GLOB.hairstyles_list)
		var/datum/sprite_accessory/facial_hair/picked_beard = pick(GLOB.facial_hairstyles_list)
		to_chat(exposed_human, "<span class='notice'>Hair starts sprouting from your scalp.</span>")
		exposed_human.hairstyle = picked_hair
		exposed_human.facial_hairstyle = picked_beard
		exposed_human.update_hair()
```

Reply:
I prefer concise to verbose, but I'll do it.